substrate/bin/node/runtime/src/lib.rs

```diff
@@ -1766,6 +1773,51 @@ impl pallet_asset_conversion::Config for Runtime {
 	type BenchmarkHelper = ();
 }
 
+pub type NativeAndAssetsFreezer =
+	UnionOf<Balances, AssetsFreezer, NativeFromLeft, NativeOrWithId<u32>, AccountId>;
+
+/// Benchmark Helper
+#[cfg(feature = "runtime-benchmarks")]
+pub struct AssetRewardsBenchmarkHelper;
+
+#[cfg(feature = "runtime-benchmarks")]
+impl pallet_asset_rewards::benchmarking::BenchmarkHelper<NativeOrWithId<u32>>
+	for AssetRewardsBenchmarkHelper
+{
+	fn staked_asset() -> NativeOrWithId<u32> {
+		NativeOrWithId::<u32>::WithId(100)
+	}
+	fn reward_asset() -> NativeOrWithId<u32> {
+		NativeOrWithId::<u32>::WithId(101)
+	}
+}
+
+parameter_types! {
+	pub const StakingRewardsPalletId: PalletId = PalletId(*b"py/stkrd");
+	pub const CreationHoldReason: RuntimeHoldReason =
+		RuntimeHoldReason::AssetRewards(pallet_asset_rewards::HoldReason::PoolCreation);
+}
+
+impl pallet_asset_rewards::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type RuntimeFreezeReason = RuntimeFreezeReason;
+	type AssetId = NativeOrWithId<u32>;
+	type Balance = Balance;
+	type Assets = NativeAndAssets;
+	type PalletId = StakingRewardsPalletId;
+	type CreatePoolOrigin = EnsureSigned<AccountId>;
+	type WeightInfo = ();
+	type AssetsFreezer = NativeAndAssetsFreezer;
+	type Consideration = HoldConsideration<
+		AccountId,
+		Balances,
+		CreationHoldReason,
+		LinearStoragePrice<ExistentialDeposit, ConstU128<0>, Balance>,
```

Comment:
same I think the price for items and bytes are a bit inconsistent.

`deposit` function makes some assumption about per-item and per-byte price (15 cents and 6 cents).

and here we make a fixed deposit and ignore the number of items and number of bytes.

maybe it should be unified into:
```
standard_per_item_price = 15 CENTS
standard_per_bytes_price = 6 CENTS
fn deposit(items, bytes) { make use of the constants above }
struct StandardFootprintPrice;
impl Convert for StandardFoorprintPrice { make use of the constants above }
```

WDYT?

Reply:
Yes it is good for the scope of this PR.

As note I think the story about pricing footprint is not very complete. on one hand we have a `Footprint` type which give some information about the storage usage, and then in the runtime we price the consideration regardless of the footprint information with some hardcoded deposit which will have to be updated every time the size of the value stored is changing.

I feel like instead we should have a standard pricing of footprint, with maybe some discount for some specific consideration.

Anyway, this would be for another PR.